packages/conf-openblas/conf-openblas.0.2.0/files/centos_install.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+yes | yum update
+yes | yum install git
+yes | yum install gcc gcc-c++ gcc-gfortran make kernel-devel
+
+git clone https://github.com/xianyi/OpenBLAS.git
```

Comment:
Could we shallow-clone a specific version of OpenBLAS?

Reply:
Sure

Reply:
For CentOS, we don't need to build from source, takes way to long time. We can just install from binary. Check this Dockerfile https://github.com/owlbarn/owl/blob/6de0d5aafa85a179bbfcc8234a6cb77f50b2a617/docker/Dockerfile.centos#L15

Reply:
So we can probably remove this sh file, but you may also need to install epel-release

Reply:
I think a depext on epel-release is not a problem, I would imagine lots of packages needing it anyway